Doppler.HtmlEditorApi/Controllers/CampaignsController.cs

```diff
@@ -173,6 +178,7 @@ public async Task<IActionResult> CreateCampaignContentFromTemplate(string accoun
                     HtmlHead: head,
                     Meta: unlayerTemplateData.Meta,
                     PreviewImage: unlayerTemplateData.PreviewImage,
+                    CampaignName: "",
```

Comment:
For consistency:

```suggestion
                    CampaignName: campaignContent.campaignName,
```

Reply:
Add a new commit https://github.com/FromDoppler/doppler-html-editor-api/pull/166/commits/3bd943ca49cc2f612ddaabc3742245de90d52554, here we get the name of the template